backend/app/main.py

```diff
@@ -2,6 +2,8 @@
 
 import argparse
 
+from typing import Dict, Any
+
```

Comment:
I think it is pylint that suggests using one line per import. For example, this would be split into two lines. Just something to consider in the future.

Reply:
We may need a quick discussion on this as I prefer this on different lines but without the from typing on each line.
i.e.
Not
from typing import Dict
from typing import Any

but instead:-
from typing import (
    Dict,
    Any
)

As then it is easier to see the grouping of imports. But that may just be me. It would be nice if we all did the same thing but not vital or important, hence I have not said anything until now. This is especially the case for longer lists. If that list could be in alphabetical order even better, it makes finding where methods comes from easier.

Reply:
Interesting. I have not seen this syntax before. I personally prefer it. I think it would be interesting to see what pylint thinks about it. I usually do the former but I would do the latter if it is generally accepted as the correct syntax in the community.